app_dart/lib/src/model/appengine/task.dart

```diff
@@ -262,27 +262,77 @@ class Task extends Model<int> {
     _status = value;
   }
 
-  /// Update [Task] related fields based on LUCI's [BuildPushMessage].
-  String updateFromBuildPushMessage(BuildPushMessage pushMessage) {
-    if (pushMessage.build?.status == Status.started) {
+  /// Update [Task] fields based on a LUCI [Build].
+  void updateFromBuild(Build build) {
+    final List<String>? tags = build.tags;
+    // Example tag: build_address:luci.flutter.prod/Linux Cocoon/271
```

Comment:
Is the tags the `Tags` on the build page: https://ci.chromium.org/ui/p/flutter/builders/prod/Mac%20framework_tests_widgets/3643/overview?

Reply:
No, it's seems to be specific to the push messages (I was able to verify this from the GCP logs)